src/Commands/MigrateDumpCommand.php

```diff
@@ -451,8 +451,13 @@ private static function sqliteDataDump(array $db_config, string $schema_sql_path
         $tables = preg_split('/\s+/', implode(' ', $output));
 
         foreach ($tables as $table) {
+            // We don't want to dump the migrations table here
+            if ('migrations' === $table) {
+                continue;
+            }
+
             // Only migrations should dump data with schema.
```

Comment:
this comment seems like a copy-paste, I don't think it applies here

Reply:
Yup was a copy/paste. Fixed!